### ISSUE TITLE
Broken example with settings

### ISSUE THREAD

**Issue by darky** [tiaod/moleculer-io#17]
This code on 1.0.8 no more works:

```
broker.createService({
  name: 'io',
  mixins: [SocketIOService],
  settings: {
    port: 3000 //will call initSocketIO() on broker.start()
  }
})
broker.start()
```

Need to add `server: true` for proper listening on port:

```
broker.createService({
  name: 'io',
  mixins: [SocketIOService],
  settings: {
    port: 3000, //will call initSocketIO() on broker.start()
    server: true
  }
})
broker.start()
```

**Comment by tiaod** [comment]
Sorry, I forgot to add the `server: true` to the default options. 😅

**Comment by tiaod** [comment]
Fixed at 1.0.9
